server/models/alumniSchema.js

```diff
@@ -2,6 +2,21 @@ var mongoose = require('mongoose')
 
 const Schema = mongoose.Schema;
 
+/*
+  All Time Availabilities are to be stored in GMT+0 timezone
+  Example record
+  {
+    id: 'Sunday-400',
+    day: 'Sunday',
+    time: 400
+  }
+*/
+const timeAvailabilitySchema = new Schema({
```

Comment:
Is there a reason why this needs to be a Schema rather than a js object?
A schema is a mongoose datatype used to store objects in a collection; unlike requests, availabilities only interacts with one model so it doesn't need to be stored separately

Reply:
Best practices for schemas are also to give each one a separate file so that there isn't additional hunting required - this would probably also be a good idea for timeAvailability